tests/testthat/test-pin-read-write.R

```diff
@@ -1,5 +1,6 @@
 test_that("can round trip all types", {
   skip_if_not_installed("qs")
+  skip_if_not_installed("arrow")
```

Comment:
We didn't find this with `R-CMD-check-hard` because it's already skipping the test (qs not installed). Just a limitation of that current check. 🤷‍♀️